File: core/src/main/java/org/kobjects/krash/api/Anchor.java
package org.kobjects.krash.api;

public interface Anchor {

  /**
   * Returns the normalized width of the view. For the screen, this value is negative.
   */
  float getWidth();

  /**
   * Returns the normalized height of the view. For the screen, this value is negative.
   */
  float getHeight();

  Object getTag();
}

File: core/src/main/java/org/kobjects/krash/api/Anchor.kt
package org.kobjects.krash.api

interface Anchor {
    /**
     * Returns the normalized width of the view. For the screen, this value is negative.
     */
    val width: Float

    /**
     * Returns the normalized height of the view. For the screen, this value is negative.
     */
    val height: Float
    val tag: Any?
}